### ISSUE TITLE
Uncaught Error: Cannot find module '../../../../../node_modules/url/url.js'

### ISSUE THREAD

**Issue by butane** [atom/atom#14550]
### Steps to reproduce:

1. Clone the atom repository and checkout "v1.17.0" tag.
2. Run "./script/build"
3. After build process is complete run atom. "./out/atom-1.17.0-amd64/atom --safe"
4. This error message pops up as soon as the editor window opens up.

**NOTE:** Above steps worked for v1.16.0 on the same machine without any issues.

**Atom**: 1.17.0 x64
**Electron**: 1.3.15
**OS**: Ubuntu 14.04.2
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: Cannot find module '../../../../../node_modules/url/url.js'

```
At module.js:457

Error: Cannot find module '../../../../../node_modules/url/url.js'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (/home/vivekd/Desktop/atom/out/atom-1.17.0-amd64/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/home/vivekd/Desktop/atom/out/atom-1.17.0-amd64/resources/app/src/module-cache.js:354:1)
    at Module.require (home/vivekd/Desktop/atom/out/atom-1.17.0-amd64/resources/app/static/index.js:40:43)
    at require (internal/module.js:20:19)
    at customRequire (/home/vivekd/Desktop/atom/out/atom-1.17.0-amd64/resources/app/static/<embedded>:73:1)
    at get_url (/home/vivekd/Desktop/atom/out/atom-1.17.0-amd64/resources/app/src/default-directory-provider.js:13:1)
    at DefaultDirectoryProvider.module.exports.DefaultDirectoryProvider.directoryForURISync (/home/vivekd/Desktop/atom/out/atom-1.17.0-amd64/resources/app/src/default-directory-provider.js:22:1)
    at Project.module.exports.Project.getDirectoryForProjectPath (/home/vivekd/Desktop/atom/out/atom-1.17.0-amd64/resources/app/src/project.js:295:1)
    at AtomEnvironment.module.exports.AtomEnvironment.openLocations (/home/vivekd/Desktop/atom/out/atom-1.17.0-amd64/resources/app/src/atom-environment.js:1309:1)
    at EventEmitter.outerCallback (/home/vivekd/Desktop/atom/out/atom-1.17.0-amd64/resources/app/src/application-delegate.js:251:1)
    at emitThree (events.js:121:20)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
./script/build
./out/atom-1.17.0-amd64/atom --safe
```

### Non-Core Packages

```
None (Opened in safe mode)
```

**Comment by rsese** [comment]
@butane - do you get the same error if you run `script/clean` before building again?

**Comment by sr-wisptools-vivek** [comment]
@rsese Yes, I got the same error after running `script/clean` and then building again.

Additionally, I tried removing the `.atom` and `.config/Atom` folders, but it did not make a difference.

**Note**
- The error message pops up once when Atom is opened.
- When I create a new empty file (File -> New File) there are **no** errors. But when I try to save the file the same error occurs and fails to save.
- When I try to open an existing file, this error occurs and fails to open the file.
- When I try to toggle the tree view, this error occurs and fails to display the tree view.
- I tried opening a bunch of other windows (Timecop, Deprecation cop etc..) and the same error repeats.

**Comment by butane** [comment]
@rsese Yes, I got the same error after running `script/clean` and then building again.

Additionally, I tried removing the `.atom` and `.config/Atom` folders, but it did not make a difference.

**Note**
- The error message pops up once when Atom is opened.
- When I create a new empty file (File -> New File) there are **no** errors. But when I try to save the file the same error occurs and fails to save.
- When I try to open an existing file, this error occurs and fails to open the file.
- When I try to toggle the tree view, this error occurs and fails to display the tree view.
- I tried opening a bunch of other windows (Timecop, Deprecation cop etc..) and the same error repeats.
